server/server.py

```diff
@@ -2,12 +2,15 @@
 	import socketserver
 	import time
 	import sqlite3
+	import argparse
+	from ipaddress import ip_address
+	from sys import argv
```

Comment:
your right, I at one point removed that but this branch got screwed up (you can see it in the graphs->network)